lib/ansible/modules/windows/win_timezone.ps1

```diff
@@ -19,28 +19,32 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
-$params = Parse-Args $args;
-
-$result = New-Object psobject @{
-    win_timezone = New-Object psobject
+$result = @{
     changed = $false
+    win_timezone = @{}
 }
 
-$timezone = Get-Attr -obj $params -name timezone -failifempty $true -resultobj $result
+$params = Parse-Args $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+
+$timezone = Get-AnsibleParam -obj $params -name "timezone" -failifempty $true -resultobj $result
```

Comment:
Do we want to change this and remove the -resultobj to keep in line with the rest of the modules?